app/helpers/grant_submissions/questions_helper.rb

```diff
@@ -0,0 +1,5 @@
+module GrantSubmissions::QuestionsHelper
+  def get_question_id(question)
+    question.id ||= 0
```

Comment:
this actually modifies the ID. if you just wanted to get the ID or a 0, it should be `question.id || 0`